bigquery_etl/cli/query.py

```diff
@@ -582,7 +582,9 @@ def backfill(
         sys.exit(1)
 
     query_files = paths_matching_name_pattern(name, sql_dir, project_id)
+    print("These are the query files: " + str(query_files))
     if query_files == []:
+        print("Running generators!!!!!")
```

Comment:
Is this print adding much value? It seems `generate_all` already logs this:
https://github.com/mozilla/bigquery-etl/blob/05792e0a08c511a75edafa84573fb5f37b663ea9/bigquery_etl/cli/generate.py#L76

Reply:
Only for testing purposes